requirements.txt

```
@@ -1,2 +1,3 @@
 python-dotenv>=0.13.0,<=0.13.99
 spotipy>=2.19.0,<=2.19.99
+tqdm>=4.64.1
```

```suggestion
tqdm>=4.64.1,<=4.64.99
```

Would recommend limiting package version to be the latest patch version as done with previous packages -- we don't want it to suddenly upgrade to say v5.0.0 and break functionality.